src/universal/components/Dashboard/DashLayout.js

```diff
@@ -0,0 +1,64 @@
+import React, {PropTypes} from 'react';
+import look, {StyleSheet} from 'react-look';
+
+import Helmet from 'react-helmet';
+import {head} from 'universal/utils/clientOptions';
+
+import NotificationBar from 'universal/components/NotificationBar/NotificationBar';
+
+let styles = {};
+
+const DashLayout = (props) => {
```

Comment:
right now I think our lint rules allow for both, because I have definitely added some code that doesn't have em. You & Terry like the parens, it's the eslint default, I say we stick with that, I'll add a rule.